source/whats-new.txt

```diff
@@ -23,6 +26,64 @@ Learn what's new in:
 * :ref:`Version 3.7 <version-3.7>`
 * :ref:`Version 3.6 <version-3.6>`
 
+
+.. _version-4.10:
+
+What's New in 4.10
+------------------
+
+New features of the 4.10 {+driver-short+} release include:
+
+- Callback Deprecation
+
+  - Callbacks are now deprecated in favor of Promises. Callbacks will
+    be removed in the next major release. The Node driver team recommends
+    migrating to promises where possible:
+
+    - Use ``async/await`` syntax.
+    - Use the Node.js `callbackify utility <https://nodejs.org/api/util.html#utilcallbackifyoriginal>`__
+
+      .. code-block:: js
+
+         require('util').callbackify(() => collection.findOne())(callback)
+
+    - Use `then <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/then>`__ syntax
+
+      .. code-block:: js
+
+         collection.findOne().then(res => callback(null, res), err => callback(err))
+
+  - If you are unable to migrate to promises in a large codebase, you can
+    use the {+node-legacy+} package.
+
+To learn more, see `v4.10.0 Release Highlights <https://github.com/mongodb/node-mongodb-native/releases/tag/v4.10.0>`__.
+
+.. _version-4.9:
+
+What's New in 4.9
+-----------------
+
+New features of the 4.9 {+driver-short+} release include:
+
+- Fixed an inconsistency with ``writeConcern`` options in the type definitions.
+- Included the latest BSON release, which adds automatic UUID support. See the
+  BSON release notes `here <https://github.com/mongodb/js-bson/releases/tag/v4.7.0>`__.
+
+To learn more, see `v4.9.0 Release Highlights <https://github.com/mongodb/node-mongodb-native/releases/tag/v4.9.0>`__.
+
+.. _version-4.8:
+
+What's New in 4.8
+-----------------
+
+New features of the 4.8 {+driver-short+} release include:
+
+- Added auto-completion and type safety for nested keys in an update filter
+- ``client.startSession()`` can now be called before connecting to MongoDB
+- ``estimatedDocumentCount()`` method can now accept a comment
```

Comment:
these are copied over from previous release notes (they weren't included in master for some reason, likely directly targeted the branch). Since this isn't meant as a backport, we can create another ticket to make these punctuation changes and backport them as a separate item.